components/board/genericlinux/board.go

```diff
@@ -236,8 +250,11 @@ func (b *sysfsBoard) AnalogReaderByName(name string) (board.AnalogReader, bool)
 }
 
 func (b *sysfsBoard) DigitalInterruptByName(name string) (board.DigitalInterrupt, bool) {
-	b.logger.Warn("Digital interrupts are not currently supported on sysfs boards.")
-	return nil, false
+	interrupt, ok := b.interrupts[name]
+	if !ok {
+		return nil, false
+	}
+	return interrupt.interrupt, true
```

Comment:
I don't like the name duplication here. There are 3 structs called interrupts: a) the one from the third-party library that actually implements these things, the `board.DigitalInterrupt` interface that gets returned from this function, and the `genericlinux.digitalInterrupt` struct which stores pairs of the other two to associate them together. I'd prefer to rename at least that last one, but don't have a good name. Any suggestions?